Source/Fuse.PushNotifications/Android/Impl.uno

```diff
@@ -335,50 +333,53 @@ namespace Fuse.PushNotifications
 				}
 				else if (notificationStyle == BIGPICTURESTYLE || notificationStyle.equals(BIGPICTURESTYLE))
 				{
+					NotificationCompat.BigPictureStyle style = new NotificationCompat.BigPictureStyle();
+
+					if (bigTitle!=null && !bigTitle.isEmpty())
+						style.setBigContentTitle(bigTitle);
+					if (bigBody!=null && !bigBody.isEmpty())
+						style.setSummaryText(bigBody);
+
 					if (featuredImage.startsWith("http://") || featuredImage.startsWith("https://"))
 					{
-						BigPictureStyleHttp bps = new BigPictureStyleHttp(notificationManager, id, notificationBuilder, bigTitle, bigBody, sound);
+						BigPictureStyleHttp bps = new BigPictureStyleHttp(notificationManager, id, notificationBuilder, style, sound);
 						bps.execute(featuredImage);
 						return;
 					}
 					else
 					{
-						Bitmap bitmap = null;
 						int iconResourceID = com.fuse.R.get(featuredImage);
 
 						if (iconResourceID!=-1)
 						{
-							bitmap = android.graphics.BitmapFactory.decodeResource(context.getResources(), iconResourceID);
+							style.bigPicture(android.graphics.BitmapFactory.decodeResource(context.getResources(), iconResourceID));
 						}
 						else
 						{
 							String packageName = "@(Project.Name)";
```

Comment:
Looks like this one is unused now...

Reply:
Why @kusma?

Reply:
Hmm, seems I messed up, sorry about that.